packages/design-tokens/src/tokens/functional/components/bento/colors.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  brand: {
+    Bento: {
+      background: {
+        value: 'var(--base-color-scale-white-0)',
+        dark: 'var(--base-color-scale-gray-9)',
```

Comment:
This should be the same as we use [in Card](https://github.com/primer/brand/blob/main/packages/design-tokens/src/tokens/functional/components/card/colors.json#L5): 

```suggestion
        value: 'var(--brand-color-canvas-default)',
        dark: 'var(--brand-color-canvas-subtle)',
```